exonum/src/events/network.rs

```diff
@@ -45,6 +45,22 @@ pub enum ConnectedPeerAddr {
     Out(String, SocketAddr),
 }
 
+impl ConnectedPeerAddr {
+    pub fn is_incoming(&self) -> bool {
+        match self {
+            ConnectedPeerAddr::In(_) => true,
+            ConnectedPeerAddr::Out(_, _) => false,
+        }
+    }
+
+    pub fn is_outgoing(&self) -> bool {
+        match self {
```

Comment:
Maybe `!self.is_incoming()` is better.

Reply:
Maybe it is ok here, because this method is not used.  
But in general  pedantic clippy does not allow to use `if !x {} else {}` it requires you to reshuffle it and it is part of the standard library style where you have for ex. `is_some()` and `is_none()` for `Option`